spring-boot/src/test/java/org/dcsa/conformance/manual/ManualEBLSIOnlyTest.java

```diff
@@ -0,0 +1,86 @@
+package org.dcsa.conformance.manual;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+class ManualEBLSIOnlyTest extends ManualTestBase {
+
+  public ManualEBLSIOnlyTest() {
+    super(log); // Make sure no log lines are logged with the Base class logger
+  }
+
+  @Test
+  void testManualEBLFlowFirstScenario() {
+    getAllSandboxes();
+    getAvailableStandards();
+
+    SandboxConfig sandbox1 =
+        createSandbox(
+            new Sandbox(
+                "Ebl",
+                "3.0.0",
+                "Conformance SI-only",
+                "Carrier",
+                true,
+                "Ebl-SI - Carrier testing: orchestrator"));
+
+    SandboxConfig sandbox2 =
+        createSandbox(
+            new Sandbox(
+                "Ebl",
+                "3.0.0",
+                "Conformance SI-only",
+                "Carrier",
+                false,
+                "Ebl-SI - Carrier testing: synthetic carrier as tested party"));
+
+    updateSandboxConfigBeforeStarting(sandbox1, sandbox2);
+    List<ScenarioDigest> sandbox1Digests = getScenarioDigests(sandbox1.sandboxId());
+    assertEquals(3, sandbox1Digests.size());
+    assertTrue(sandbox1Digests.getFirst().scenarios().size() >= 9);
+
+    updateSandboxConfigBeforeStarting(sandbox2, sandbox1);
+    List<ScenarioDigest> sandbox2Digests = getScenarioDigests(sandbox2.sandboxId());
+    assertTrue(sandbox2Digests.isEmpty());
+
+    // Run all tests on: Supported shipment types scenarios
+    for (int i = 0; i < 9; i++) {
```

Comment:
As just discussed and investigated together: I'll add a method to verify all the returned digests are conformant. That method can not be turned on yet, because the other subgroups needs to be added still.